generators/base/generator-base.mjs

```diff
@@ -395,12 +362,7 @@ export default class JHipsterBaseGenerator extends PrivateBase {
    * @param {string} translationKeyMenu - i18n key for entry in the menu
    */
   addElementToMenu(routerName, iconName, enableTranslation, clientFramework, translationKeyMenu = _.camelCase(routerName)) {
```

Comment:
Move to the client generator.

Reply:
I will, for sure :-)

Reply:
I've tried, this one is quite hard to achieve (harder than it seems). I'll do it later in another round (and sorry for that)